schemas/spdm.json

```diff
@@ -0,0 +1,30 @@
+{
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "definitions": {
+        "Responder": {
+            "title": "Spdm tcp responder configuration",
+            "description": "The configuration used to add remote spdm responder to the system",
+            "type": "object",
+            "properties": {
+                "Hostname": {
+                    "description": "Hostname or IP of spdm tcp responder",
+                    "type": "string"
+                },
+                "Port": {
+                    "description": "Network port spdm tcp responder is listening on",
+                    "type": "number"
+                },
+                "Name": {
+                    "description": "The name of the object",
+                    "type": "string"
+                },
+                "Type": {
+                    "description": "The type of spdm tcp responder",
+                    "enum": ["Responder"]
+                }
+            },
+            "required": ["Hostname", "Port", "Type"]
+        }
```

Comment:
@spinler - the Interface name will be `SpdmTcpResponder` with that in context do we really need another property called `Type` - which is hardcoded to `Responder` ?